foundations/foundation-common/src/main/java/io/servicecomb/foundation/common/base/ServiceCombConstants.java

```diff
@@ -49,4 +49,12 @@
   String CONFIG_CSE_PREFIX = "cse.";
 
   String CONFIG_KEY_SPLITER = "_";
+
+  String CONFIG_MICROSERVICE_DEVELOPMENT_FRAMEWORK_KEY = "framework.name";
+
+  String DEFAULT_MICROSERVICE_DEVELOPMENT_FRAMEWORK = "UNKNOWN";
+
+  String CONFIG_MICROSERVCIE_DEVELOPMENT_FRAMEWORK_VERSION_KEY = "framework.version";
+
+  String CONFIG_MICROSERVICE_REGISTER_WAY_KEY = "registreBy";
```

Comment:
It should be registeredBy

Reply:
Sorry, the spelling is wrong here,it has been changed to registerBy,it's sure better to use registeredBy, but this field is provided by SC, i have to align with them...